ci/integration/deploy-pipeline.yaml

```diff
@@ -100,6 +101,7 @@ spec:
                 --set "connector.entityID=${CONNECTOR_ENTITY_ID}" \
                 --set "connector.metadataURL=${CONNECTOR_METADATA_URL}" \
                 --set "connector.metadataSigningTruststoreBase64=${CONNECTOR_METADATA_SIGNING_TRUSTSTORE_BASE64}" \
+                --set "translator.connectorNodeNationalityCode"=${CONNECTOR_CODE_COUNTRY_CODE} \
```

Comment:
Why pass `CONNECTOR_CODE_COUNTRY_CODE` as a param to this stage of the deployment instead of hard coding here? Also, elsewhere i've used `NATIONALITY_CODE` to be consistent with the eIDAS spec, might be worth changing here.

Reply:
Because Phil said so 🤷‍♂ 
You're right - it should be `NATIONALITY_CODE ` - I've corrected that now